rare/game_launch_helper/lgd_helper.py

```diff
@@ -0,0 +1,152 @@
+import os
+import platform
+import shutil
+from dataclasses import dataclass
+from logging import getLogger
+from typing import List, Tuple
+
+from PyQt5.QtCore import QProcess, QProcessEnvironment
+from legendary.core import LegendaryCore
+from legendary.models.game import InstalledGame, LaunchParameters
+
+logger = getLogger("Helper")
+
+
+@dataclass
+class LaunchArgs:
+    app_name: str
+    offline: bool = False
+    skip_version_check: bool = False
+    wine_prefix: str = ""
+    wine_bin: str = ""
+
+
+@dataclass
+class LaunchReturnValue:
```

Comment:
The name of this class makes it look like it is meant to hold the return value of the launched game which doesn't make much sense as concept tbh, but maybe a better name could be found. Not much better still but maybe ProcessParams or LaunchParams?